llvm/test/CodeGen/RISCV/rvv/vl-opt-no-prop.ll

```diff
@@ -0,0 +1,80 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 2
```

Comment:
These three test files seem to have valuable coverage regardless of which approach we take towards reducing VL.  Can you commit these separately and rebase?